### DISCUSSION TITLE
Improve user experience on /articles

### DISCUSSION THREAD

**Discussion by Olaleye-Blessing** [codu-code/codu#583] [Ideas]
## Context
Currently, users have to scroll back up in order to select a topic or filter the article by `newest` | `oldest` | `top`. This will be annoying especially if they have loaded a good number of articles.

## Expected Behavior
There should be an easy way to either select their topic of interest or filter the topics.

### Current behavior
<img width="1436" alt="Screenshot 2023-10-21 at 10 42 26" src="https://github.com/codu-code/codu/assets/70102539/4d6e31b9-e418-42fe-8711-345ffdb5f953">

### Expected behavior
<img width="1432" alt="Screenshot 2023-10-21 at 10 42 05" src="https://github.com/codu-code/codu/assets/70102539/7023c2e5-a08b-4b0c-bfa3-b4ed2a91471a">

## Steps to reproduce

1. Go to the [articles page](https://www.codu.co/articles?filter=newest)
2. Scroll down to load more articles
3. Notice the `filter` dropdown and `recommended topics` disappeared

## Additional info
I will like to work on it.

[a comment by an automated account is not shown]

**Comment by NiallJoeMaher** [comment]
Thanks for the input! 🦾  Do you have any examples of sites doing this so I can see what it might look like?

**Comment by Bitnagar** [comment]
@NiallJoeMaher I think what @Olaleye-Blessing is asking is something similar to [firstissue.dev](https://www.firstissue.dev/)

**Comment by NiallJoeMaher** [comment]
@Bitnagar From a quick glance, it doesn't seem to work too well on my mobile and takes up a lot of space 🙈

**Comment by Olaleye-Blessing** [comment]
Not concrete examples but [MDN](https://developer.mozilla.org/en-US/docs/Learn) has a sample; `(View on PC)` You could access any of the page content from the `in this article` section cause it sticks to the screen. You don't need to scroll back up to do this.

<img width="1467" alt="Screenshot 2023-10-21 at 12 24 36" src="https://github.com/codu-code/codu/assets/70102539/5bdc4089-31e9-41e5-9187-faec0f0b1645">

#

I made a video of what it will look like: [https://streamable.com/zqow4j](https://streamable.com/zqow4j)

**Comment by Olaleye-Blessing** [comment]
I added the mobile version here: [https://streamable.com/bhcccm](https://streamable.com/bhcccm)

**Reply by Olaleye-Blessing** [reply]
IMO, the mobile version shown here is not bad. Does it look like it takes up a lot of space? @NiallJoeMaher

**Comment by Olaleye-Blessing** [comment]
> @NiallJoeMaher I think what @Olaleye-Blessing is asking is something similar to [firstissue.dev](https://www.firstissue.dev/)

Yea, exactly. Thanks for the perfect example.

**Comment by Bitnagar** [comment]
@NiallJoeMaher my bad. I saw it on desktop. It was fine. Forget to check for mobile! 😅 Apologise.

**Comment by NiallJoeMaher** [comment]
@Bitnagar not a problem at all! Good idea, and we had another similar discussion recently. Just seeing what the most eloquent way of tackling something like this (by stealing from some other good designs) 🙈
